### DISCUSSION TITLE
Hold skip buttons for bigger amount of time skipped

### DISCUSSION THREAD

**Discussion by governos** [PaulWoitaschek/Voice#2023] [Ideas]
A bigger jump is handy for unchaptered recordings, where the seeking bar is imprecise.

The skip amount setting can be used for the user to define the bigger skip amount as well.

Thank you for your development. Your application continues to improve whilst remaining elegant.

EDIT: Oh sorry to have made an issue, I missed the discussions.

**Comment by Uno-Muno** [comment]
Absolutely agreed
